### ISSUE TITLE
After SPV3 deployment, crc all files for failures

### ISSUE THREAD

**Issue by BinToss** [yumiris/HCE#205]
Then prompt to retry deployment.

**Comment by yumiris** [comment]
Iterate through the manifest and compare the size declared for each file with the actual size of the file on the filesystem.

**Comment by BinToss** [comment]
Agreed. The hardest part of this system is getting enough time and energy
to dedicate to its creation.
